packages/backend/src/models/SearchModel/index.ts

```diff
@@ -72,6 +73,17 @@ export class SearchModel {
                 `${DashboardsTableName}.name`,
                 `${DashboardsTableName}.description`,
                 { spaceUuid: 'space_uuid' },
+                {
+                    validationErrors: this.database.raw(`
```

Comment:
I don't really know how the performance of this is going to be and whether postgres will:
1. filter everything first and then run this subquery on only a small number of rows
2. first compute this field (loads of lookups on another table) and then filter afterwards

To be safe, could we move this to another query in the same transaction so it would go:
1. Do search to find the relevant charts
2. Do seperate query to get validation errors for returned chart ids from step 1
3. Return them together